watch-package.js

```diff
@@ -36,11 +44,47 @@ module.exports = function watchPackage(pkgDir, exit) {
   stdin.stderr = through()
   stdin.stdout = through()
 
+  if (typeof taskName !== 'undefined' && taskName.trim() !== '') {
+    if (!pkg.scripts[taskName]) {
```

Comment:
I was thinking about in between lines 47&48 so after we already checked for undefined. 
But I will have to pull down the code before deciding on this, so will get back to you after the weekend